atd-vzd/metadata/databases/default/tables/public_atd_txdot_crashes.yaml

```diff
@@ -245,6 +245,8 @@ insert_permissions:
         - milepoint_2
         - mpo_id
         - nbr_of_lane
+        - near_trauma_center_distance
+        - near_trauma_center_id
```

Comment:
Another question for @patrickm02L, will these new fields be shown in the UI and be editable? I'm thinking that, if the answer is no or if we don't know, then maybe we shouldn't add insert/update permissions yet? I also acknowledge that we aren't in a great state as far as existing permissions matching what the UI does...

Reply:
Yeah so based off Patricks comment it sounds like i could remove Editors from accessing the changes tab, and then in that case i could also not add insert/update/delete permissions for editors yet. I am also wondering about whether will these fields be shown in the UI and/or be editable @patrickm02L

Reply:
@Rose, yes please turn off permissions for Editors. No direction for editing in the UI right now, but I can follow up with Xavier - which will probably be a separate Issue.